src/pages/NewSchedule.tsx

```diff
@@ -8,35 +11,75 @@ import { NickNameForm } from '../components/features/CreateScheduleForm/NickName
 import { ScheduleInputForm } from '../components/features/CreateScheduleForm/ScheduleInputForm';
 
 export const NewSchedule = () => {
-  const { Funnel, setStep } = useFunnel(newScheduleStepNames);
+  const accessToken = localStorage.getItem('accessToken');
+  const { uuid } = useParams();
 
-  const [newSchedule, setNewSchedule] = useState<newSchedule>({
+  const navigate = useNavigate();
+  const { Funnel, step, setStep } = useFunnel(newScheduleStepNames);
+
+  useEffect(() => {
+    if (accessToken) {
+      setStep('일정입력');
+    }
+  }, [accessToken, setStep]);
+
+  const [nickName, setNickName] = useState('');
+  const [schedule, setSchedule] = useState<newSchedule>({
     dateOfScheduleList: [],
-    nickName: '',
   });
 
-  const updateSchedule = (key: keyof newSchedule, value: string | Schedule[]) => {
-    setNewSchedule((prev) => ({ ...prev, [key]: value }));
+  const updateDateOfScheduleList = (value: Schedule[]) => {
+    setSchedule((prev) => ({ ...prev, dateOfScheduleList: value }));
+  };
+
+  const createScheduleMutation = useMutation({
+    mutationFn: async ({ data, uuid }: { data: newSchedule; uuid: string }) => {
+      if (accessToken) {
+        return mutations.memberSchedule.createMemberSchedule.mutationFn(data, uuid);
+      } else {
+        return mutations.nonMemberSchedule.createNonMemberSchedule.mutationFn(data, uuid);
+      }
```

Comment:
혹시 여기를 이렇게 작성하면 문제가 되려나요? 이렇게 바꾸는 것이 가능하다면 api 함수가 인수를 객체로 받도록 변경하거나, 아래 52라인 `mutate({data, uuid})` 부분을 `mutate(data, uuid)`로 변경해햘 것 같아요

```suggestion
    mutationFn: accessToken ? mutations.memberSchedule.createMemberSchedule.mutationFn : mutations.nonMemberSchedule.createNonMemberSchedule.mutationFn;
      }
```

Reply:
아뇨! `63e75b6` 수정했습니다아!